test/releaseNotes.test.js

```diff
@@ -0,0 +1,37 @@
+'use strict'
+
+const tap = require('tap')
+
+const { getPrNumbersFromReleaseNotes } = require('../src/utils/releaseNotes')
+
+tap.test('Should return the correct PR numbers', async () => {
+  const testReleaseNotes =
+    "## What's Changed\n" +
```

Comment:
let's use a template string or even load from a file

Reply:
I have tried to turn that into a template string, but then the markdown parser perceived the whole release notes as a single `code_block` token, which resulted in incorrect output.

Reply:
that would be really weird because a template string is equivalent to concatenating string with new lines, except that the latter is syntactically uglier

Reply:
My bad. The problem was that I have also removed the `\n` characters thinking that the markdown parser will pick up the line breaks from the template string.
Test string is now turned into a template string.